commitlint.config.js

```diff
@@ -1 +1,25 @@
-module.exports = { extends: ['@commitlint/config-conventional'] };
+module.exports = {
```

Comment:
Add `// @see https://github.com/conventional-changelog/commitlint/blob/v12.0.1/@commitlint/config-conventional/index.js` here for easy reference / comparison access.

Not sure how to automate upgrading this URL when we update deps.